streams/src/test/java/org/apache/kafka/streams/processor/internals/GlobalStreamThreadTest.java

```diff
@@ -221,7 +221,6 @@ public void shouldTransitionToRunningOnStart() throws Exception {
         globalStreamThread.shutdown();
     }
 
-    @Test
```

Comment:
Why this?

Reply:
Whoops, not on purpose. Thanks for the check